gsl_projet/views.py

```diff
@@ -227,3 +249,15 @@ def _get_is_one_field_active(self, field_names):
             if self.request.GET.get(field_name) not in (None, ""):
                 return True
         return False
+
+    DS_STATE_MAPPINGS = {key: value for key, value in Dossier.DS_STATE_VALUES}
+
+    def _get_status_placeholder(self):
+        if self.request.GET.get("status") in (None, "", []):
+            return "Tous"
+        return ", ".join(
+            [
+                self.DS_STATE_MAPPINGS[status]
+                for status in self.request.GET.getlist("status")
+            ]
+        )
```

Comment:
`join` marche pour n'importe quel itérable, pas besoin de caster en liste d'abord : 

```suggestion
        return ", ".join(
            self.DS_STATE_MAPPINGS[status]
            for status in self.request.GET.getlist("status")
        )
```

Reply:
Je m'interroge sur le contrôle de l'input utilisateur, ici. Est-ce qu'on a une erreur 500 si on <del>pourrit</del> pollue les paramètres GET avec des valeurs <del>pétées</del> fantaisistes ?

L'utilisation du mapping me semble éviter les problèmes d'injection XSS, donc OK de ce côté-là.

Reply:
C'est bon. :)